src/Microsoft.ML.Onnx/ProtabilityCatalog.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.Model.Onnx;
+using Microsoft.ML.UniversalModelFormat.Onnx;
+
+namespace Microsoft.ML
+{
+    public static class ProtabilityCatalog
+    {
+        /// <summary>
+        /// Convert the specified <see cref="ITransformer"/> to ONNX format. Note that ONNX uses Google's Protobuf so the returned value is a Protobuf object.
+        /// </summary>
+        /// <param name="catalog">A field in <see cref="MLContext"/> which this function associated with.</param>
+        /// <param name="transform">The <see cref="ITransformer"/> that will be converted into ONNX format.</param>
+        /// <param name="inputData">The input of the specified transform.</param>
+        /// <returns></returns>
+        public static ModelProto ConvertToOnnx(this ModelOperationsCatalog.PortabilityTransforms catalog, ITransformer transform, IDataView inputData)
+        {
+            var env = new MLContext(seed: 1);
+            var ctx = new OnnxContextImpl(env, "model", "ML.NET", "0", 0, "com.microsoft", OnnxVersion.Stable);
```

Comment:
>com.microsoft [](start = 75, length = 13)

I am not an expert on ONNX, but is this some sort of namespace? When this was functionality that was meant for internal Microsoft consumption, having this be a default may have been appropriate, but when shipped as ML.NET (which involves, of course, many non-Microsoft), should it not be something the user specifies? Or is its role something I don't fully appreciate? (Could be that `com.microsoft` is entirely possible here, I'd just like to understand why if so, because that seems odd. If it is appropriate, surely a comment to that effect would be desirable.)

Reply:
It should be "ai.onnx.ml". Sorry about causing confusion. Currently, we don't have anything Microsoft-specific.

---
In reply to: [245428777](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245428777) [](ancestors = 245428777)

Reply:
No idea, since I have no idea what it is for. 😄 But perhaps there are experts elsewhere we can ask to decide on the right answer.

Reply:
Because ML.NET is not Microsoft-specific, I just change it to `machinelearning.dotnet` so that `machinelearning.dotnet` + `ML.NET converter version` + `model name` can be unique IDs of models. 

---
In reply to: [245446247](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245446247) [](ancestors = 245446247)